h/static/scripts/tests/controllers/search-bar-controller-test.js

```diff
@@ -9,9 +10,13 @@ describe('SearchBarController', function () {
   var dropdown;
   var dropdownItems;
   var ctrl;
+  var clock;
 
   before(function () {
-    template = '<input data-ref="searchBarInput">' +
+    template = 
+      '<div class="search-bar__lozenges" data-ref="searchBarLozenges"></div>' +
```

Comment:
You can use template strings now to make writing templates over multiple lines easier. Have a look at `form-controller-test.js`